### DISCUSSION TITLE
When is Update 5.0.0?

### DISCUSSION THREAD

**Discussion by Frillaut** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3283] [Q&A] [answered]
Hi!
Is it possible to find out when the 5.0.0 update will be?
Some controls are needed (NumericUpDown, TextBox with AutoSuggestion). Therefore, I think to wait for the update or try to do it yourself.

**Comment by nicolaihenriksen** [comment, marked as answer]
@Frillaut Until there is an official release, you can use the [nightly preview builds](https://www.nuget.org/packages/MaterialDesignThemes/5.0.0-ci330#versions-body-tab). At the time of writing, the latest preview is [5.0.0-ci330](https://www.nuget.org/packages/MaterialDesignThemes/5.0.0-ci330)

@Keboo is the maintainer who decides when to push an official release; I am not sure when he plans to do so.

However, the two features you refer to are still PRs that have not been merged, so they are not currently part of the preview builds.

**Comment by Keboo** [comment]
@nicolaihenriksen is correct. The basic plan for the releases is when they are ready I kick them out. However for the 5.0.0 release there are lots of breaking changes so I am working on updates some of my own apps to the 5.0.0 release as a means of testing it.
